macaw-ppc/src/Data/Macaw/PPC/Disassemble.hs

```diff
@@ -225,12 +199,12 @@ disassembleFn :: (ppc ~ SP.AnyPPC var, PPCArchConstraints var)
               -- ^ The initial registers
               -> Int
               -- ^ Maximum size of the block (a safeguard)
-              -> ST s ([Block ppc ids], Int, Maybe String)
+              -> ST s (Block ppc ids, Int)
 disassembleFn _ lookupSemantics nonceGen startAddr regState maxSize = do
   mr <- ET.runExceptT (unDisM (tryDisassembleBlock lookupSemantics nonceGen startAddr regState maxSize))
   case mr of
-    Left (blocks, off, exn) -> return (blocks, off, Just (show exn))
-    Right (blocks, bytes) -> return (blocks, bytes, Nothing)
+    Left (blocks, off, _exn) -> return (blocks, off)
```

Comment:
I'd like to find a way to propagate this exception information up (assuming there is ever useful information there).  I'm actually not sure right now what could error out.

Reply:
We could do it inline by adding a new Block type that essentially just contained the address and the error.  Then any analysis that attempted to execute that block could deliver a more focused result.

We can also just log it (when the logging gets here).

Reply:
Added https://github.com/GaloisInc/macaw/issues/67

Reply:
Are you looking for Macaw to immediately print out an error message?  If not, you can terminate the block with `TranslationError` and look for that.  If you want more structured information, then you could create an architecture-specific terminal statement.
I removed the error message because the block itself contained the error in the form of a translation error.

Reply:
Actually, looking into it the implementation of `failAt` already turns the thrown exception into a `TranslationError` and that reported exception is extraneous.